core/services/workflows/engine_test.go

```diff
@@ -591,6 +591,7 @@ func TestEngine_ResumesPendingExecutions(t *testing.T) {
 	)
 	err = eng.Start(ctx)
 	require.NoError(t, err)
+	defer eng.Close()
```

Comment:
You could use the helper for these simple ones:
```suggestion
	servicetest.Run(t, eng)
```

Reply:
Added :)